### DISCUSSION TITLE
动态展示能否增加一个按照关注列表里的分类展示呢？

### DISCUSSION THREAD

**Discussion by lrs95** [the1812/Bilibili-Evolved#3068] [功能建议 / Ideas]
动态展示能否增加一个按照关注列表里的分类展示呢？我关注的有两百多个up主，各种类别的都有，我都在关注列表进行了分类，但是在动态里直接就全部展示出来了，感觉很杂乱。因为关注的人多，常常会看不到我真真想看的消息，如果可以按关注分类展示动态更新，然后在分类里再详细按照投稿或专栏或追番追剧进行细分，我想这样体验会更好。不知道能否实现这样的功能

**Comment by the1812** [comment]
我实现不了, 需要 b 站官方支持

**Comment by Geeyun-JY3** [comment]
推荐脚本：https://greasyfork.org/scripts/396032

部分重复：#1279
